tests/unit/db/test_api.py

```diff
@@ -39,6 +39,8 @@ def test_create_service_definition(mock_session, mock_uuid):
     mock_session.return_value.__enter__.return_value.add.assert_called_once
 
 
+"""
+TODO
```

Comment:
This section is commented out.  Please remove it now and add back later when it is working.

Reply:
Done